src/__test__/unit/reducers/DataStorageReducer.spec.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-undef */
+import reducer, { initialState } from '../../../reducers/datastorageReducer';
+import * as actions from '../../../constants/actions';
+
+describe('data storage reducer', () => {
+  it('should return the initial state', () => {
+    expect(reducer(initialState, {})).toEqual(initialState);
+  });
+
+  it(`should handel ${actions.SET_ID}`, () => {
```

Comment:
```suggestion
  it(`should handle ${actions.SET_ID}`, () => {
```

The same typo is in the descriptions of the other two test in this file too.